docs/reference/modules/indices/circuit_breaker.asciidoc

```diff
@@ -177,3 +177,64 @@ it triggers the regex circuit breaker and returns an error.
 +
 {es} only applies this limit if
 <<script-painless-regex-enabled,`script.painless.regex.enabled`>> is `limited`.
+
+[[eql-circuit-breaker]]
```

Comment:
Nice! I see though that the EQL and ML circuit breakers were simply copied over to the Circuit Breakers page.

As mentioned on https://github.com/elastic/elasticsearch/issues/85851 it would be nice to move them indeed to the Circuit Breakers, but then add links from their original places to these new sections here. So in EQL original page for example you can replace the original text with a paragraph saying `See EQL circuit breakers <<eql-circuit-breaker,here>>`. That way we will not need to maintain two copies of the text.

I wonder what to do with the paragraph keys. We cannot have `[[eql-circuit-breaker]]` in two places. Probably you can remove the key from the original paragraph and leave it in the new copied paragraph.